src/DataLakeStore/DataLakeStore/help/Restore-AzDataLakeStoreDeletedItem.md

```diff
@@ -0,0 +1,182 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataLakeStore.dll-Help.xml
+Module Name: Az.DataLakeStore
+ms.assetid: D231E9A0-DC1E-411B-A87A-56A8C767F6C5
+online version: https://docs.microsoft.com/en-us/powershell/module/az.datalakestore/restore-azdatalakestoredeleteditem
+schema: 2.0.0
```

Comment:
Please regenerate this file using https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md. This should fix your CI failure.